core/src/main/java/com/sequenceiq/cloudbreak/controller/ClusterController.java

```diff
@@ -124,6 +124,23 @@
     public ResponseEntity<String> updateCluster(@PathVariable Long stackId, @RequestBody UpdateClusterJson updateJson) throws CloudbreakSecuritySetupException {
         Stack stack = stackService.get(stackId);
         MDCBuilder.buildMdcContext(stack);
+        if (updateJson.getUserNamePasswordJson() != null) {
```

Comment:
I would refactor 'updateJson.getUserNamePasswordJson()' to an individual variable because it is used many times and the could would be much easeir to read.